server/.flaskenv

```diff
@@ -0,0 +1,2 @@
+FLASK_APP=server.py
+FLASK_DEBUG=1
```

Comment:
Is "flaskenv" the flask preferred env file naming? We might also not want to debug by default, might be better commented out or left in the readme to inform devs how to get the server into a debug state if that's what this is for.

Where will "FLASK_APP" be used?

Reply:
.flaskenv is specifically for environment variables for flask I believe, such as FLASK_APP, FLASK_ENV, FLASK_DEBUG, etc. 
Can I ask why we won't need debug by default? if not debug then it will be in production mode...shouldn't it only be in production mode when it is in production? 
I think FLASK_APP is only used to indicate how to load the application when using the ```flask run``` command. Since we are using ```python3 server.py``` command to run it, then it might be unnecessary. I will remove it.

Reply:
yeah so DEBUG off by default is probably the better way to go (not 100% sure what flask does with it) but most "production" modes end up being more performant. If we want to create a `.flaskenv` locally for ourselves, we could add `FLASK_DEBUG=1` on our machines, but it shouldn't be a default imo. 

We can also create a `.sample-flaskenv` and ignore `.flaskenv` so the sample can provide the fields we will need ourselves. but `.env` files themselves typically should be ignored becuase every environment might be a little different.

Reply:
This is what DEBUG does: "By enabling debug mode, the server will automatically reload if code changes, and will show an interactive debugger in the browser if an error occurs during a request." Okay, I can remove the FLASK_DEBUG line from the .flaskenv file and note it in the README if you would like